src/chunk.c

```diff
@@ -2178,6 +2189,9 @@ get_chunks_in_time_range(Hypertable *ht, int64 older_than, int64 newer_than, Mem
 	end_strategy = (older_than == PG_INT64_MAX) ? InvalidStrategy : BTLessStrategyNumber;
 	time_dim = hyperspace_get_open_dimension(ht->space, 0);
 
+	if (time_dim == NULL)
+		time_dim = hyperspace_get_closed_dimension(ht->space, 0);
```

Comment:
aren't we replacing one assumption for another here? Should we add proper check in case this fails too?

Reply:
added an `Ensure` to guarantee we have either open or close time dimension defined for the hypertable